### ISSUE TITLE
Owncast's custom javascript is blocked

### ISSUE THREAD

**Issue by AruMoon** [uBlockOrigin/uAssets#22226]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://watch.owncast.online/
https://ety.cybre.stream/
https://livestream.ds-server.nl/
https://radio.nogood.studio/
Or any other owncast instance. You can pick any from Directory: https://directory.owncast.online/
```

### Description

uBlock is blocing customjavascript for any Owncast instance. It's not good, because custom javascript may contain essential code by streamer for some integrations or plugins
It's blocked by this filter somehow
```
/^https?:\/\/[^/]+\/[a-z0-9]{15,}(\?hope=.*)?$/$script,from=~coco.fr|~greenmangaming.com|~ign.com|~justjared.com|~live-tennis.eu|~myorderbox.com|~namemc.com|~novinky.cz|~o2.pl|~onet.pl|~realtor.ca|~regway.com|~roblox.com|~sap.com|~spongemc.org|~spongepowered.org|~studio-svet.ru|~stumbleupon.com|~trueachievements.com|~tv.yandex.by|~tv.yandex.kz|~tv.yandex.ru|~wetteronline.de|~wp.pl
```

### Other extensions used

SponsorBlock, DuckDuckGo Privacy Essentials, Firefox Color, To Google Translate

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.55.0
Firefox: 122
filterset (summary):
 network: 133872
 cosmetic: 66368
 scriptlet: 20272
 html: 2072
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 37434-126, 49m Δ
  ublock-badware: 7800-10, 49m Δ
  ublock-privacy: 988-0, 49m Δ
  ublock-unbreak: 2244-0, 49m Δ
  ublock-quick-fixes: 133-0, 49m Δ
  easylist: 79817-661, 49m Δ
  easyprivacy: 43563-857, 49m Δ
  urlhaus-1: 8550-0, 16h.28m
  plowe-0: 3782-1, 12d.11h.45m
  RUS-0: 40209-126, 1h.16m Δ
filterset (user): [empty]
trustedset:
 added: [array of 3 redacted]
switchRuleset:
 added: [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 418 ms (selfie)
 maxAssetCacheWait: 170 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 1
 network:
  owncast.online: 1
```
</details>

**Comment by dimisa-RUAdList** [comment]
Fix: https://github.com/easylist/ruadlist/commit/b396118af3017d368698c66e7e4014ae37b45b8f

**Comment by D4niloMR** [comment]
Btw, I don't know if that filter is 100% correct, as I don't know much about regex

I edited it, I think it's more correct that way (since the negative lookahead is typed wrong, and typed correctly it does not work)

@Yuki2718 Sorry but can you take a look?

**Comment by dimisa-RUAdList** [comment]
@D4niloMR 
> Btw, I don't know if that filter is 100% correct, as I don't know much about regex

Oh, really. In its current form, the regular expression does nothing. However, it was added to RU AdList more than 10 years ago and has long ceased to catch the scripts it was originally aimed at. In addition, in MV3 such rules will not work in principle, so I removed it: https://github.com/easylist/ruadlist/commit/4361cc9c7db60b76e84a02c5dc105773559f5cc5
